ckan/tests/logic/action/test_create.py

```diff
@@ -470,23 +477,36 @@ def test_mimetype_by_user(self):
         assert_equals(mimetype, 'application/csv')
 
     def test_mimetype_by_upload(self):
-        pass
+        test_file = file('/home/vagrant/test.txt', 'rb')
```

Comment:
could use ckan source file? or just make a string and make a file obj based on that, not sure yet